### PR TITLE
[MLIR][TOSA] Update CustomOp input and output names

### PR DESCRIPTION
**Pull request by Jerry-Ge**
Update input name from input to input_list
Update output name from outputs to output_list

To match specification: https://www.mlplatform.org/tosa/tosa_spec.html#_custom

### CODE REVIEW

File: mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td
Comment (CoTinker) [inline mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td:1]: > `inputs` is the set of tensor inputs to the custom operator.
> `outputs is the list of tensors returned by the operator.

Please update the description, too.

**Review by CoTinker — CHANGES_REQUESTED** [review]
little comment.

File: mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td
Comment (Jerry-Ge) [reply mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td:1]: thanks for noting this and the review! updated the description.

**Review by CoTinker — APPROVED** [review]
LGTM, thanks.

**Pull request merged**